### PR TITLE
[FIX #227] Added check to insure message count is greater than zero

### LINKED ISSUE #227: nats-bench panics when message amount is 0
```
go run nats-bench.go -n 0 msg.test
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x5f9940, 0xc042008070)
        C:/Go/src/runtime/panic.go:500 +0x1af
main.main()
        D:/workspace/go/src/github.com/nats-io/nats/examples/nats-bench.go:76 +0x68e
exit status 2
```

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

**Comment by kozlovic** [comment]
LGTM

File: examples/nats-bench.go [hunk 1]
```diff
@@ -47,6 +47,10 @@ func main() {
 		usage()
 	}
 
+	if *numMsgs < 1 {
```
Comment (derekcollison) [inline examples/nats-bench.go]: <=0?

Comment (derekcollison) [inline examples/nats-bench.go]: Does it have to be >0?

Comment (aricart) [inline examples/nats-bench.go]: Yes - test is to abort run if not 1 or more.

File: examples/nats-bench.go [hunk 2]
```diff
@@ -47,6 +47,10 @@ func main() {
 		usage()
 	}
 
+	if *numMsgs <= 0 {
```
Comment (zquestz) [inline examples/nats-bench.go]: NIT: For what it is worth. I find *numMsgs < 1 much easier to read.

**Pull request merged**